helm/values.yaml

```diff
@@ -244,7 +244,6 @@ applications:
 
 api:
   enabled: true
-  upgrader: false
```

Comment:
Maybe keep it as a comment to keep default behaviour, but still communicate about this option.